packages/components/src/DateTimePickers/pickers/TimePicker/TimePicker.component.js

```diff
@@ -76,20 +76,17 @@ export class TimePicker extends React.Component {
 			this.scrollItemIntoView(this.props.textInput);
 		}
 	}
-	componentDidUpdate(prevProps) {
```

Comment:
Remove this because TimePicker will not update. Usually it will create a new instance when toggle it.